docs/model_library/Hydraulics/index.rst

```diff
@@ -205,4 +216,141 @@ Allows pumping only if a pump exists in a pipeline.
         \textcolor{red}{C_{l,\tilde{l}}^{Pump}} = \textcolor{green}{\nu^{Pump}} \cdot \textcolor{red}{y_{l,\tilde{l},[t]}^{Pump}}
         + \textcolor{green}{\nu^{Electricity}} \cdot \textcolor{green}{\rho.g} \cdot \sum_{t \in T}\textcolor{red}{H_{l,\tilde{l},t}^{Pump}} \cdot \textcolor{red}{F_{l,\tilde{l},t}^{Piped}}
 
+    
+**Linearizing the Equations/constraints specific to the co_optimize method**
+
+    **Parameters**
+    :math:`\textcolor{green}{\Delta_I}` =                        Length of interval of flows for building piecewise linear model
+    :math:`M` =                       Large enough constant
+
+    **Binary Variables**
+
+    :math:`\textcolor{red}{z_{l,\tilde{l},t, i}},i\in \{0,1,2,...,\{0,1,2,..., \lceil 70000/\textcolor{green}{\Delta_I}\rceil - 1}` =     Intermediate binary variables to determine the section of the piecewise linear graph.
+
+
+    **Continuous Variables**
+
+    :math:`\textcolor{red}{\lambda_{l,\tilde{l},t,i}}, i\in \{0,1,2,..., \lceil 70000/\textcolor{green}{\Delta_I}\rceil\}` =         Intermediate continuous variables, convex combination multipliers.
+
+    :math:`\textcolor{red}{term_{l,\tilde{l},t}}` =                      Continuous variables for flow raised to the power 1.85 (in RHS of Hazen-Williams).
+
+    :math:`\textcolor{red}{term2_{l,\tilde{l},t, d}}` =                  Continuous variables for the product of pressure drop and binary to select diameter (in LHS of Hazen-Williams).
+
+    :math:`\textcolor{red}{ec_{l,\tilde{l},t}}` =                      Node pressure
+
+    :math:`\textcolor{red}{Z^{Hydraulics}}` =                   Continuous variables for electricity cost for a particular pump at particular time period.
+
+
+    **Effective diameter rule:** :math:`\forall \textcolor{blue}{l,\tilde{l} \in LLA}`
+
+    We remove this constraint. When effective diameter is required, we would use the expression of binaries.
+
+    **Setting term in RHS:** :math:`\forall \textcolor{blue}{l,\tilde{l} \in LLA}, \textcolor{blue}{t \in T}`
+
+    Set the right hand side term flow to the power 1.85 in Hazen-Williams equation.
+
+    .. math::
+
+        \textcolor{red}{F_{l,\tilde{l},t}^{Piped}}
+        = \sum_{i \in \{0,1,2,..,\lceil 70000/\textcolor{green}{\Delta_I} \rceil\}}i \cdot \textcolor{green}{\Delta_I} \cdot \textcolor{red}{\lambda_{l, \tilde{l}, t, i}}
+
+    .. math::
+
+        \textcolor{red}{term_{l,\tilde{l},t}}
+        = \sum_{i \in \{0,1,2,..,\lceil 70000/\textcolor{green}{\Delta_I} \rceil\}}(1.84E-6 \cdot i \cdot \textcolor{green}{\Delta_I})^{1.85} \cdot \textcolor{red}{\lambda_{l,\tilde{l},t, i}}
```

Comment:
Is the number in this equation supposed to be 1.84 x 10^-6? I.e., scientific notation? Please format so it looks better than this: 
![image](https://github.com/project-pareto/project-pareto/assets/26798258/f7f8a687-e10a-469b-957d-316530cd9c68)

Reply:
Thanks, I have updated to the scientific notation.